interface/src/app/tribe_client/tribe_client.js

```diff
@@ -1,42 +1,45 @@
 angular.module('adage.tribe_client', [
-    'adage.tribe_client.resource',
-    'adage.tribe_client.directives',
+  'adage.tribe_client.resource',
+  'adage.tribe_client.directives',
+  'adage.tribe_client.genesets'
 ])
 
-.factory( 'UserFactory', ['User', function( User ) {
-    var promise = null;
-    var user = null;
+.factory('UserFactory', ['User', function(User) {
```

Comment:
Yep, exactly, re-formatting thanks to our new JS standard. I almost did not include this file in the pull request, but decided to do so because of the injecting of 'adage.tribe_client.genesets' above, which is necessary for this pull request.